cairo_fuzz/fuzz/src/main.rs

```diff
@@ -68,13 +57,9 @@ pub fn cairo_fuzz_run(
         cairo_runner.finalize_segments(&mut vm)?;
     }
     if secure_run {
-        verify_secure_runner(&cairo_runner, true, &mut vm)?;
-    }
-    cairo_runner.relocate(&mut vm)?;
-
-    if cairo_run_config.print_output {
-        write_output(&mut cairo_runner, &mut vm)?;
+        verify_secure_runner(&cairo_runner, true, None, &mut vm)?;
```

Comment:
Why is the `print_output` option removed?

Reply:
There is no `print_output` field anymore. There is still a `write_output` function that I removed by mistake. I'll add it!